production/helm/loki/templates/read/statefulset-read.yaml

```
@@ -102,8 +102,10 @@ spec:
           {{- end }}
           securityContext:
             {{- toYaml .Values.loki.containerSecurityContext | nindent 12 }}
+          {{- with .Values.loki.readinessProbe }}
           readinessProbe:
-            {{- toYaml .Values.loki.readinessProbe | nindent 12 }}
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
           {{- if .Values.read.livenessProbe }}
           livenessProbe:
             {{- toYaml .Values.read.livenessProbe | nindent 12 }}
```

Just noticed: This is odd, for unknown reasons, loki read has an own livenessProbe.

WDYT?

```suggestion
          {{- with .Values.read.livenessProbe | default .Values.loki.livenessProbe }}
          livenessProbe:
            {{- toYaml . | nindent 12 }}
```

---

Yeah, I noticed that too. But I think it belongs in another PR.

---

Will fix in another PR.